### ISSUE TITLE
'shuffle-salts' corrupts wp-config.php if semi-colon in original salt

### ISSUE THREAD

**Issue by cldevs** [wp-cli/wp-config-transformer#17]
Summary:
"wp config shuffle-salts" creates a malformed wp-config.php file if a semi-colon is contained in the original salt

Steps to reproduce:
Place a semi-colon in one of the salts in wp-config.php and run "wp config shuffle-salts"

Severity - High (Breaks site as it is unable to load wp-config.php)

**Comment by schlessera** [comment]
_**Additional comment from @cldevs from #15:**_

Again, the issue is that a semi-colon somewhere in the string of a value fools the transformer into thinking that's the end of the key-value declaration which results in a corrupt wp-config.php file and winds up taking down a Wordpress site.

`WPConfigTransformer.php - Line 199-203`
```php
$contents = preg_replace(
   sprintf( '/(?<=^|;|<?php\s|<?\s)(\s*?)%s/m', preg_quote( trim( $old_src ), '/' ) ),
   '$1' . str_replace( '$', '$', trim( $new_src ) ),
   $this->wp_config_src
);
```

Is this just a matter of including the ending newline character at the end of each search string (e.g. after the semi-colon) so it matches right to the end of the line instead of stopping part-way? Then including a newline character at the end of the replace? Any other implications here?

**Comment by schlessera** [comment]
@fjarrett Any specific thoughts on what the best approach is here? As we already had a similar fix, there's the chance we're circling around...

**Comment by fjarrett** [comment]
@schlessera I followed the steps above and I cannot reproduce this issue.

**Comment by schlessera** [comment]
@fjarrett I cannot replicate either with the latest release (v2.2.0).

@cldevs Is this still an issue for you? I think it might have been fixed in the mean time but the fix was only included with the latest release.

**Comment by cldevs** [comment]
Confirmed this is now fixed in v2.2.0!  Thanks!
